Source/FolioReaderContainer.swift

```diff
@@ -165,4 +153,35 @@ public class FolioReaderContainer: UIViewController {
     override public func preferredStatusBarStyle() -> UIStatusBarStyle {
         return isNight(.LightContent, .Default)
     }
+
+	// MARK: - Helpers
+
+	public static func setUpConfig(config: FolioReaderConfig, epubPath path: String) {
```

Comment:
`shouldRemoveEpub` need to be added to this setup public, so it can be used in this custom init. Consider same approach of presentReader with default value.

`public static func setUpConfig(config: FolioReaderConfig, epubPath path: String, shouldRemoveEpub: Bool = true)`